guest-os-info.md

```diff
@@ -0,0 +1,34 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``kubevirt.io/os-short-id`` :
```

Comment:
As far as I understand we are going to use `os-short-id` not only for VMs. Please provide information on other places as well.

Reply:
Right, so we might want to move this to a more general section and provide links from Virtual Machine section in the user guide and in the future also from offline virtual machines and templates if needed.
@fabiand  any thoughts ?

Reply:
For sure you can apply them wherever you like, but as far as I understand this, this is about the use in combination with vm-presets, and presets only care about vms (if you apply it on a template it has no meaning for a preset).

Reply:
@rmohr  we also thought about adding the label to offline VMs (with a combination of presets if needed) , but this was not finalized yet.
This could be changed in the future if we do so, and keep the current PR as is for now.